help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -1420,12 +1420,37 @@
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
-      <h4>Logix</h4>
+      <h4 id="common_logix_action">Logix</h4>
 
-      <p>LogixNG has an action called Logix that emulates how JMRI Logix works. The action Logix
-      has one expression and a number of boolean actions. A boolean action is an action that takes
-      a boolean value (true/false) and then decides what to do. The action Logix was added mainly
-      to support import of JMRI Logix to LogixNG.</p>
+      <p>LogixNG has an action called <strong>Logix</strong> that emulates how JMRI Logix works. The
+      action Logix has one expression and a <a href="#boolean_logix_action">Logix Action</a>.  The
+      digital boolean <strong>Many</strong> action can be used to support multiple
+      <strong>Logix Actions</strong> as shown in the example.  A Logix Action responds to
+      <strong>True</strong>, <strong>False</strong>, or <strong>Either</strong> Logix evaluation
+      states.  The action Logix was added mainly to support import of JMRI Logix to LogixNG.</p>
+
+      <div style="margin-left: 2em;">
+        <a href="images/chapter5/other_logix_action.png"><img class="image-border" src=
+        "images/chapter5/other_logix_action.png" alt=
+        "Chapter 5 other logix action"></a>
+      </div>
+
+      <p>The Logix action supports <strong>On Change</strong> and <strong>When Triggered</strong>.</p>
+
+      <p>An example Logix within LogixNG.  This is a tweaked import from Logix.  The <strong>Log data</strong> actions
+      were added and the individual <strong>True</strong> LogixNG actions were combined within a
+      standard Many.</p>
```

Comment:
This sentence looks strange to me:

The `Log data` actions were added and the individual `True` LogixNG actions were combined within a standard Many.

Reply:
That sentence describes the tweaks (changes) that were done after the Logix Import.

Reply:
I see. Thank you.